topics/metagenomics/tutorials/metagenomics-binning/turorial.md

```diff
@@ -0,0 +1,418 @@
+---
+layout: tutorial_hands_on
+
+title: "Binning of metagenomic sequencing data"
+zenodo_link: "https://zenodo.org/record/7818827"
+zenodo_link_results: "https://zenodo.org/record/7845138" 
+level: Introductory
+questions:
+  - "What is metagenomic binning refers to?"
+  - "Which tools should be used for metagenomic binning?"
+  - "How to assess the quality of metagenomic data binning?"
+# objectives:
```

Comment:
Can you add objectives, keypoints and time estimation?

Reply:
Objectives, keypoints, and time estimaton have been added.